tests/test_interface.rb

```diff
@@ -253,7 +295,11 @@ def validate_ipv4_proxy_arp(inttype_h)
       cmd = show_cmd(interface.name)
 
       # puts "value - #{v[:proxy_arp]}"
-      pattern = (/^\s+ip proxy-arp/)
+      if node.client.api == 'NXAPI'
+        pattern = /^\s+ip proxy-arp/
+      elsif node.client.api == 'gRPC'
+        pattern = /^\s+proxy-arp/
```

Comment:
Agreed - however I also want to look at some ways to refactor the YAML to avoid having to specify the same string in config_get_token, config_set, and test_config_get_regex - mind if I defer this to a later commit?